build.sh

```diff
@@ -30,11 +30,20 @@ then
   for api in $apis
   do 
     [[ -d "$api" ]] && apidir=$api || apidir=apis/$api
-    dotnet sln AllProjects.sln add $apidir/*/*.csproj
-  
+    for project in $apidir/*/*.csproj
```

Comment:
It was not possible using the old style projects. Trying to add them caused an error. But now that the new style projects work, we can add them, so I've removed these changes.